.github/workflows/zip.yml

```diff
@@ -41,3 +43,16 @@ jobs:
         # Zip the entire output directory since the builder adds subdirectories we don't know the
         # name of.
         path: zip_output_dir
+    - name: Install gcloud tool
+      if: ${{ always() }}
+      run: scripts/install_access_gcloud.sh
+    - name: Access gcloud account
+      if: ${{ always() }}
```

Comment:
I think we can omit the `if` and it should always run - I could be wrong though.

Reply:
I think it would only run the rest if previous steps successfully passed. 
GHA have[ a doc ](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions)about if successful/failed steps